packages/server/modules/workspaces/tests/integration/graph/query.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { expect } from 'chai'
+import cryptoRandomString from 'crypto-random-string'
+import {
+  createTestContext,
+  testApolloServer,
+  TestApolloServer
+} from '@/test/graphqlHelper'
+import {
+  BasicTestUser,
+  createAuthTokenForUser,
+  createTestUser
+} from '@/test/authHelper'
+import { AllScopes, Roles } from '@speckle/shared'
+import {
+  CreateWorkspaceDocument,
+  GetWorkspaceDocument
+} from '@/test/graphql/generated/graphql'
+
+describe('Root Query type', () => {
+  let apollo: TestApolloServer
+
+  const testUser: BasicTestUser = {
+    id: '',
+    name: 'John Speckle',
+    email: 'root-query-type-user@example.org',
+    role: Roles.Server.Admin
+  }
+
+  before(async () => {
+    await createTestUser(testUser)
```

Comment:
These kind of integration tests running GQL operations and DB queries definitely have an overhead, so I would definitely merge all of these into a single test file, to avoid these being slower than they have to be.

Even outside of that, the division seems weird - they're not split by workflows/domains but by they location they're in in the GQL schema, which is kind of arbitrary. In `query` you test getting a single workspace, in `user` you test getting a single user's workspaces, surely these could all just be in a "workspace retrieval" test file?

I'd go even further and do all core CRUD tests in the same test file